go/cmd/dolt/cli/arg_parser_helpers.go

```diff
@@ -117,6 +117,8 @@ const (
 	ListFlag         = "list"
 	UserParam        = "user"
 	NoPrettyFlag     = "no-pretty"
+	AddIgnoredFlag   = "add-ignored"
+	ShowIgnoredFlag  = "show-ignored"
```

Comment:
Strive to always match Git's command arguments unless there's a really compelling reason not to. It's frustrating to users who know git (or who are just searching for git help online to try and figure out dolt) if we only partially follow git's cli syntax. 

For `git add` I would expect this to be `dolt add --force` or `dolt add -f` to match git's syntax. 

For `git status` I would expect this to be `dolt status --ignored` (with no shorthand option).

Reply:
Don't worry, for `dolt add`, the command is `dolt add --force`

Somehow, I had convinced myself that `git commit --force` was a command for staging ignored files during commit, but because we already created a `dolt commit --force` I had to make a new flag. Turns out, there's no flag in git for staging ignored files and I'm not sure why I was so convinced that there was.

`add-ignored` has been removed.

For `git status` I had gone with `show-ignored` because I thought that `git status ignored` required a value and didn't want to implement the same flag name with different semantics. I've switched it to be just `--ignored`.